### ISSUE TITLE
publish new rust version to crates.io

### ISSUE THREAD

**Issue by mpetri** [huggingface/tokenizers#688]
The rust code of the tokenizers and the respective python bindings have evolved quite substantially since the last version has been published to crates.io.

Is it possible to publish a new version to crates.io to enable using the new features via a standard crates.io import instant of specifying the git repo directly?

**Comment by yfernando-bw** [comment]
+1. 

I created this a while ago for the same thing https://github.com/huggingface/tokenizers/issues/677

**Comment by n1t0** [comment]
Indeed, we haven't published a new version for a very long time. The reason is that we currently have a dependency on a git repository (cf https://github.com/huggingface/tokenizers/blob/master/tokenizers/Cargo.toml#L43).

I'll probably publish a version of this crate as a quick hack so that we can publish a new version of the library.

**Comment by mpetri** [comment]
@n1t0 I wanted to follow up on this issue. It looks like there is now a version of this library on crates.io?
